src/ng-select/ng-select.component.ts

```diff
@@ -704,4 +713,4 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
         this.clearAllText = this.clearAllText || config.clearAllText;
         this.disableVirtualScroll = this.disableVirtualScroll || config.disableVirtualScroll;
     }
-}
+}
```

Comment:
Ups. Probably tslint will fail. I will add better webpack configuration to show tslint in webpack dev model, I see that it is not showing tslint warnings in the console.